udf-server-python/src/io/aklivity/zillabase/udf-server.py

```diff
@@ -16,7 +16,10 @@
 import importlib.util
 import inspect
 import os
+import pkg_resources
```

Comment:
this package is [deprecated](https://setuptools.pypa.io/en/latest/pkg_resources.html) and the [importlib.resources](https://docs.python.org/3/library/importlib.resources.html) package should be used instead